numba/np/linalg.py

```diff
@@ -611,8 +618,15 @@ def dot_3_vm(context, builder, sig, args):
     for val in m_shapes:
         check_c_int(context, builder, val)
 
-    call_xxgemv(context, builder, do_trans, mty, m_shapes, m_data,
-                v_data, out.data)
+    zero = ir.Constant(intp_t, 0)
```

Comment:
There's a preference in the code base for using the `context` where possible, e.g.
`zero = context.get_constant(types.intp, 0)`
(same applies throughout).